app/controllers/health_controller.rb

```diff
@@ -0,0 +1,13 @@
+class HealthController < ApplicationController
+  def workers
+    status = worker_health_checker.all_healthy? ? :ok : :internal_error
+
+    render json: worker_health_checker.statuses, status: status
```

Comment:
Ooops I saw this comment too late after I merged. How strongly do you feel? Would you like me to rename and namespace the controller?